src/pages/Admin/PermissionHandler.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from "react";
+import { AdminResources } from "services/types";
+import { useGetWallet } from "contexts/WalletContext";
+
+export default function PermissionHandler({
+  data,
+  children,
+}: {
+  data: AdminResources | undefined;
+  children: ReactNode;
+}) {
+  const { wallet } = useGetWallet();
+
+  return (
+    <fieldset disabled={!wallet || !data?.members.includes(wallet.address)}>
+      {children}
```

Comment:
there are some cases where sender of an admin tx need not to be a member of the multisig wallet

1. withdraws - sender might be whitelisted
2. modifications ( edit profile, fees, etc) - sender might be a delegate address

Reply:
implemented validation for deposit as per request, couldn't find delegate address attribute in endowment to connect with though @SovereignAndrey if you can help with that point?